src/pages/settings/Payments/PaymentsPage.js

```diff
@@ -110,6 +157,40 @@ class PaymentsPage extends React.Component {
         this.setState({shouldShowAddPaymentMenu: false});
     }
 
+    /**
+     * Hide the default / delete modal
+     */
+    hideDefaultDeleteMenu() {
+        this.setState({shouldShowDefaultDeleteMenu: false});
+    }
+
+    hidePasswordPrompt() {
+        this.setState({shouldShowPasswordPrompt: false});
+    }
+
+    callPasswordCallbackAndHidePopover(password) {
+        this.hidePasswordPrompt();
+        this.state.passwordFormCallback(password);
```

Comment:
Would recommend not using React `state` for anything that is not serializable. It won't cause anything to explode per se but is really unusual.

Reply:
Oops yeah good catch. This was important when you needed a password to delete accounts so different methods had to be called by the same form. That's not the case now so it doesn't need to be in the state now